src/clj/kixi/hecuba/data/sensors.clj

```diff
@@ -158,21 +166,34 @@
 
 (defn encode
   ([sensor]
-     (encode sensor false))
+   (encode sensor false))
   ([sensor remove-pk?]
-     (-> sensor
-         (cond-> (:user_metadata sensor) (user-metadata (:synthetic sensor)))
-         (cond-> remove-pk? (dissoc :device_id :sensor_id)))))
+   (-> sensor
+       (cond-> (:user_metadata sensor) (user-metadata (:synthetic sensor)))
```

Comment:
I didn't know cond-> until today, however looking at the docs you could probably put all that in one cond-> ?